src/main/java/org/quiltmc/loader/impl/transformer/ChasmInvoker.java

```diff
@@ -241,9 +238,12 @@ private static <T> T this_value_is_actually_nullable(T in) {
 	static class QuiltMetadata {
 
 		final ModLoadOption from;
+		// the dot name of this class
+		final String name;
```

Comment:
Should this be Javadoc? Eclipse doesn't show non-javadoc comments above fields as javadoc.